tests/peft_integration/test_peft_integration.py

```diff
@@ -265,17 +265,31 @@ def test_peft_add_multi_adapter(self):
                 _ = model.generate(input_ids=dummy_input)
 
                 model.set_adapter("default")
-                self.assertTrue(model.active_adapter() == "default")
+                self.assertTrue(model.active_adapter() == ["default"])
```

Comment:
Hmm, so the change is backwards incompatible for this specific case? Can we live with that?